AArch64: Failure in ContendedFieldsTests_90_1

Failure link
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal/7/

Some tests in ContendedFieldsTests_90_1 fail as shown below.
```
[2020-03-23T02:16:31.422Z] FAILED: testContentionGroupsAndClass
[2020-03-23T02:16:31.422Z] java.lang.AssertionError: org.openj9.test.contendedfields.TestClasses$ContGroupContClass calculated size 64 actual size 24 difference -40
[2020-03-23T02:16:31.422Z] 	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
[2020-03-23T02:16:31.422Z] 	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
[2020-03-23T02:16:31.422Z] 	at org.openj9.test.contendedfields.FieldUtilities.checkObjectSize(FieldUtilities.java:67)
```

--- a/runtime/port/unix/j9sysinfo.c
+++ b/runtime/port/unix/j9sysinfo.c
@@ -1675,8 +1675,15 @@ j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, const J9CacheInfoQue
 	) {
 		/* L1 data cache line size */
 		int32_t rc = (int32_t)sysconf(_SC_LEVEL1_DCACHE_LINESIZE);
-		if (rc >= 0) {
+		if (rc > 0) {
 			result = rc;
+		} else if (rc == 0) {
+			/*
+			 * Cache line size is unavailable on some systems
+			 * Use 64 as the default value because Arm Cortex ARMv8-A cores
+			 * have L1 data cache lines of that size
+			 */
+			result = 64;
 		}
 	}
 #endif

